[OCPBUGS-41344] Update minor version of RHEL 8 for worker node (#83659)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

{product-title} {product-version} clusters are available at https://console.redhat.com/openshift. With the {cluster-manager-first} application for {product-title}, you can deploy {product-title} clusters to either on-premises or cloud environments.

// Double check OP system versions

> {product-title} {product-version} is supported on {op-system-base-full} 8.6 and any subsequent version of {op-system-base-full} 8 which is released before End of Life of {product-title} {product-version} as well as on {op-system-first} 4.14.

Reviewer: ```suggestion
{product-title} {product-version} is supported on {op-system-base-full} 8.6 and a later version of {op-system-base} 8 that is released before End of Life (EoL) of {product-title} {product-version} and on {op-system-first} 4.14.
```
